frontend/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "^6.6.0",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "ajv": "^8.17.1",
```

Comment:
do you use the ajv package? if not remove it.
please take a look on the packages listed in this file and remove the unused ones.

Reply:
Done